### PR TITLE
hh-ia.a.2 hh-ia.a.8 and hh-ia.a.9 added

### PR DESCRIPTION
**Pull request by dperezBM**
Corresponding issue: [#1046](https://github.com/INSPIRE-MIF/helpdesk-validator/issues/1046)

### CODE REVIEW

File: data-hh/hh-ia/ets-hh-ia-bsxets.xml [hunk 1]
```diff
@@ -206,7 +281,7 @@ $messages)
 									</translationTemplates>
 								</TestAssertion>
 								<TestAssertion id="EID6a13c03d-7e4a-4de6-99e8-6795b77d2522">
-									<label>hh-ia.a.3: EnvHealthDeterminantTypeValue</label>
+									<label>hh-ia.a.4: EnvHealthDeterminantTypeValue</label>
```
Comment (fabiovinci) [inline data-hh/hh-ia/ets-hh-ia-bsxets.xml:284]: Is this codelist verified for the new feature types too?

EnvHealthDeterminantTypeValue <a name="EnvHealthDeterminantTypeValue"></a> | //schema-element(hh:EnvHealthDeterminantMeasure)/hh:type/@xlink:href <br> //schema-element(hh:**EnvHealthDeterminantConcentrationMeasure**)/hh:type/@xlink:href <br> //schema-element(hh:**EnvHealthDeterminantNoiseMeasure**)/hh:type/@xlink:href <br> //schema-element(hh:EnvHealthDeterminantStatisticalData)/hh:type/@xlink:href | 1 | No

**Review by fabiovinci — CHANGES_REQUESTED** [review]
Hi @dperezBM,

I put a comment inline that should be verified before merging.
Thanks

File: data-hh/hh-ia/ets-hh-ia-bsxets.xml (hunk 1 above)
Comment (dperezBM) [reply data-hh/hh-ia/ets-hh-ia-bsxets.xml:284]: I have just performed a new commit with your change so I think that the PR would take it into account when it is approved.
Could you check it again?
 https://github.com/inspire-eu-validation/ets-repository/pull/827/files

Comment (fabiovinci) [reply data-hh/hh-ia/ets-hh-ia-bsxets.xml:284]: It seems ok now.

**Review by fabiovinci — APPROVED** [review]

**Pull request merged**